src/Microsoft.AspNet.Server.Kestrel/Infrastructure/KestrelTrace.cs

```diff
@@ -39,6 +40,7 @@ static KestrelTrace()
             // ConnectionWrite: Reserved: 11
             // ConnectionWriteCallback: Reserved: 12
             // ApplicationError: Reserved: 13 - LoggerMessage.Define overload not present 
+            _connectionDisconnectedWrite = LoggerMessage.Define<long, int>(LogLevel.Debug, 14, @"Connection id ""{ConnectionId}"" write of ""{count}"" bytes to disconnected client.");
         }
```

Comment:
We also want to log errors surfaced by libuv to the write callback since we will no longer throw `IOExceptions` from `SocketOutput.WriteAsync` (though we really should have done this all along since there was never a guarantee that the call to `WriteAsync` would actually see the error before). We decided that these errors should use `LogLevel.Information`.